packages/codegen-ui-react/lib/imports/import-mapping.ts

```diff
@@ -22,6 +22,7 @@ export enum ImportSource {
   LOCAL_MODELS = '../models',
   LOCAL_SCHEMA = '../models/schema',
   UTILS = './utils',
+  AMPLIFY_API = '@aws-amplify/api',
```

Comment:
Is the preference here to import from `@aws-amplify/api` or `aws-amplify`?

Reply:
AppSync's documentation advises customers to install `@aws-amplify/api` and import from there. https://docs.aws.amazon.com/appsync/latest/devguide/building-a-client-app.html

I'll cut a task to add that package to our required packages check.